scripts/build_debug.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
When I did "make build" and executed the release binary, I was able to see some DEBUG logs. Please check the below logs. These logs are supposed to appear in release builds? Please check.

> level=DEBUG ts=2021-01-31T08:42:14Z app=device-coap msg="Thread iot-0-0 starting"
level=DEBUG ts=2021-01-31T08:42:14Z app=device-coap msg="Thread iot-0-1 starting"

Reply:
Hi Vijay, this is a CSDK behavior. We start up with maximum logging enabled until the configuration is read, at which point we switch to whatever level is configured. So the services are always a bit chatty to begin with.

Reply:
Ok Iain..